### PR TITLE
BUG: Fix imperials import from astropy.units

### LINKED ISSUE #11975: cannot import name 'imperials' from 'astropy.units'
### Description

Our spectroscopy package [radis](https://github.com/radis/radis) depends on Astropy.units. Latest build fetched astropy==4.3 
 and [failed](https://app.travis-ci.com/github/radis/radis/jobs/527018350#L1767) with the following error : 
```
AttributeError: module 'astropy.units' has no attribute 'imperial'
```

- I could find a similar error encountered by @pllim in https://github.com/astropy/astropy/pull/11204#issuecomment-821844588, which helped a lot to yield a fix, below. 
- I trace the error to the change of Imports in #11592 by @namurphy  :  

### Expected behavior

Original code worked : 
```
import astropy.units as u
u.K in (u.K, u.imperial.deg_F)
>>> cannot import name 'imperials' from 'astropy.units'
```

Now I have to update the code to 

```
import astropy.units as u
from astropy.units.imperial import deg_F 
u.K in (u.K, deg_F)
>>> True
```

If this is the expected behavior (and cleaner way to import), this Issue can be closed right away. Hopefully the title will help people that would encounter the same proper to find an easy fix.

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is to re-enable old usage of `imperials` within `astropy.units` namespace.

Fixes #11975 . This is a follow-up of #11204 and #11592 . cc @namurphy 

I will also enable cron jobs, just in case...

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label.
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label.
- [ ] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

File: astropy/units/physical.py
Comment (eteq) [inline astropy/units/physical.py:12]: It's complex enough keeping a link to the story here I think is worth it
```suggestion
from . import imperial  # Need this for backward namespace compat, see issues 11975 and 11977  # noqa
```

**Review by eteq — APPROVED** [review]
FWIW, I also double-checked this test fails *without* the fix, so this LGTM modulo my small clarification suggestion.

**Review by adrn — APPROVED** [review]

**Comment by adrn** [comment]
Good catch @namurphy! LGTM

File: astropy/units/physical.py
Comment (pllim) [reply astropy/units/physical.py]: OK, I accepted this suggestion but I don't want to rerun the CI. Please see the CI from the first commit. Thanks!

**Comment by pllim** [comment]
Cron jobs from first commit still running. I'll leave it to your good judgement on when you want to merge this. Thanks for the reviews!

**Review by mhvk — APPROVED** [review]
Nice!

**Comment by eteq** [comment]
Alright, I'm going to merge this now so that we can try to roll this into a quick 4.3.1 .  Thanks for the quick action, @pllim , and @adrn and @mhvk on the quick reviews!

**Pull request merged**
